tests/unit/utils/selection-utils-test.js

```diff
@@ -0,0 +1,83 @@
+const {module, test} = QUnit;
+
+import { comparePosition } from 'mobiledoc-kit/utils/selection-utils';
```

Comment:
`import { DIRECTION } from 'mobiledoc-kit/utils/key';`, then can use `DIRECTION.FORWARD`, etc in the tests below